docs/getting-started.md

```diff
@@ -19,6 +39,34 @@ procedure on a regular basis (e.g. via cron). We recommend executing it every
 30 minutes. This is necessary to execute maintenance tasks such as enforcing
 data retention policies according to the configured policy.
 
+Add the below code snippet to the file `prom-execute-maintenance.sh` and add the database password in place of `<PASSWORD>`:
+
+```
+#/bin/sh
+export PGPASSWORD=<PASSWORD>
+psql --host localhost --port 5432 --dbname postgres --user postgres -c 'CALL prom_api.execute_maintenance();'
+```
+
+Create an other script with the below code snippet to the file install-crontab.sh and configure the absolute path to the above script. This script will create a crontab for the above defined task:
```

Comment:
make sure to tell people to change the <ABSOLUTE PATH>

Reply:
To grant exe permissions? `chmod +x <>.sh` ?